tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py{36,37,38},flake8
+
+[testenv]
+deps = -rtest-requirements.txt
+commands =
+	python setup.py test
```

Comment:
This is the deprecated command that tox is supposed to be getting us away from. :-)

Reply:
we might need to migrate to pytest first, but that will be a separate PR